health-check/src/uyuni_health_check/loki/loki_manager.py

```diff
@@ -161,17 +161,8 @@ def check_series_in_loki(
 
 def wait_promtail_init():
     loki_url = "http://localhost:3100"
-    start_time = time.time()
-    timeout = 60
-    console.log("[bold]Waiting for Promtail to process logs")
```

Comment:
I'm still missing this log.  We're waiting here for up to 60 seconds, we should inform the user why.

Actually, are we waiting here for up to 60 seconds? Seems like this will execute once and regardless of the result from `check_series_in_loki`, we continue execution. Is that expected?